start.sh

```diff
@@ -27,20 +27,26 @@ else
     GUNICORN_RELOAD=""
 fi
 
+DOMAIN_JQ='.ALLOWED_HOSTS | . - ["127.0.0.1", "localhost", ".ngrok.io"] | if . | length == 0 then "localhost" else .[0] end'
+
 if [ -z "${ENV_JSON}" ]; then
     MYSQL_HOST=$(jq -r -c ".MYSQL_HOST | values" ${ENV_FILE})
     MYSQL_PORT=$(jq -r -c ".MYSQL_PORT | values" ${ENV_FILE})
     IS_CRON_POD=$(jq -r -c ".IS_CRON_POD | values" ${ENV_FILE})
     PTVSD_ENABLE=$(jq -r -c ".PTVSD_ENABLE | values" ${ENV_FILE})
     CRONTAB_SCHEDULE=$(jq -r -c ".CRONTAB_SCHEDULE | values" ${ENV_FILE})
     RUN_AT_TIMES=$(jq -r -c ".RUN_AT_TIMES | values" ${ENV_FILE})
+    DOMAIN=$(jq -r -c "${DOMAIN_JQ} | values" ${ENV_FILE})
+    PORT=$(jq -r -c ".LOCALHOST_PORT | values" ${ENV_FILE})
```

Comment:
I'd probably set a default for this PORT incase it isn't defined in the configuration file. Just put it up above where the other defaults are. It's only used in the localhost case anyway. I guess you could put it in the jquery too.

Reply:
How does [this commit](https://github.com/tl-its-umich-edu/my-learning-analytics/pull/761/commits/2d63c9fb54edb95c31ee13411acd8fa1992fa954) look? I tried putting the `if fi` block above this, but it was resulting in `PORT` having an empty string value.

Reply:
I think we can forget about this, if we move this from the environment to the docker-compose is should always be set on the localhost.